runtime/jvmti/jvmtiEventManagement.c

```diff
@@ -100,7 +100,7 @@ jvmtiSetEventNotificationMode(jvmtiEnv* env,
 		/* Ensure the required capabilities are enabled for the event if the event is being enabled */
 
 		if (mode == JVMTI_ENABLE) {
-			switch(event_type) {
+			switch((int)event_type) {
```

Comment:
This change seems unnecessary.

Reply:
The cast is needed to resolve a compiler error related to enums. @dipak-bagadiya Can you post specific details for the compiler error?

Reply:
event_type is of jvmtiEvent thus give compile time error

Because 'event_type' is of type 'jvmtiEvent', a compile time error occurs. 
case value '48' not in enumerated type 'jvmtiEvent' {aka 'enum jvmtiEvent'} [-Werror=switch]
VIRTUAL_THREAD_MOUNT and VIRTUAL_THREAD_UNMOUNT are extension events.